addons/im_livechat/static/src/js/im_livechat_channel_form_controller.js

```diff
@@ -0,0 +1,39 @@
+odoo.define('im_livechat.ImLivechatChannelFormController', function (require) {
+"use strict";
+
+var FormController = require('web.FormController');
+
+var ImLivechatChannelFormController = FormController.extend({
+    events: _.extend({}, FormController.prototype.events, {
+        'click .o_im_livechat_channel_form_reset1': '_onResetColors1',
+        'click .o_im_livechat_channel_form_reset2': '_onResetColors2',
+    }),
+    /**
+     * @private
+     */
+    _onResetColors1: function(){
+        this.$('[name="button_background_color"] > .o_field_color').css('background-color', "#878787");
+        this.$('[name="button_text_color"] > .o_field_color').css('background-color', "#FFFFFF");
+        const changes = {
```

Comment:
no need to create variable for that, can pass inline object to `notifyChanges` call